src/Microsoft.Framework.PackageManager/Install/AppCommandsFolderRepository.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Microsoft.Framework.PackageManager
 {
-    internal class AppCommandsFolderRepository : IAppCommandsRepository
+    public class AppCommandsFolderRepository : IAppCommandsRepository
```

Comment:
Why does it have to be _public_? If for test, how about use `InternalsVisibleTo` attribute?
https://github.com/aspnet/dnx/blob/dev/src/Microsoft.Framework.Runtime/Properties/AssemblyInfo.cs#L7